bedrock/gigabit/templates/gigabit/apply.html

```diff
@@ -139,8 +139,8 @@ <h1 class="large center">{{ _('Apply') }}</h1>
             {% endtrans %}
             </li>
             <li>
-            {% trans url_gigabit_application_form='https://drive.google.com/file/d/0B9NqWPTRsfNXNjlKT0tMVlcySlhfZmVlQTBINXVrN0FSVlI0/view' %}
-              Prepare your application using our <a href='{{ url_gigabit_application_form }}'>application form</a>.
+            {% trans url_gigabit_application_form='https://drive.google.com/file/d/0B9NqWPTRsfNXNjlKT0tMVlcySlhfZmVlQTBINXVrN0FSVlI0/view', url_faq='http://mzl.la/GCF_FAQ' %}
```

Comment:
Hi @alexgibson :) I checked with @jenntbeard , here's her response:

> the reason we're using the shortlink is because we're continuing to update the .pdf version and want to make sure it's correct.

I'll update the protocol to `https`